boot/mbed/app_enc_keys.c

```diff
@@ -69,3 +69,12 @@ const struct bootutil_key bootutil_enc_key = {
 #endif
 
 #endif
+
+#if !defined(MCUBOOT_HW_KEY) && defined(MCUBOOT_ENC_IMAGES)
+int boot_retrieve_private_key(struct bootutil_key *private_key)
+{
+    private_key = (struct bootutil_key *)&bootutil_enc_key;
```

Comment:
This line, and the ones below, doesn't actually do anything. It it just copying the address of the key to a local variable. I suspect what you actually want is for the parameter to be `struct bootutil_key **private_key`.

Reply:
fixed it now. please trigger CI

Reply:
@d3zd3z CI passed. I hope we can merge this now.